docs/api/routing.rst

```diff
@@ -394,6 +394,61 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other 
+paradigms. Consider the task of building an API for a calculator which can both add and subtract 
+two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.text = str(req.get_param_as_int('x) + req.get_param_as_int('y'))
```

Comment:
This doesn't look right. It seems that the closing quote got lost along the way: `'x` :arrow_right: `'x'`.

Reply:
Ah shoot, was trying to rush this out before a meeting. Sorry about that. Will update

Reply:
Updated